### DISCUSSION TITLE
Allow to set the color of the cross hair for Interactive Image

### DISCUSSION THREAD

**Discussion by hig-dev** [zauberzeug/nicegui#2713] [Ideas / Feature Requests]
Hello,

I would like to display dark images and use the cross hair feature of the `ui.interactive_image(cross=True)`.

**Problem:** The cross hair is almost not visible because it is hard coded to be black.

**Reference:** 
https://github.com/zauberzeug/nicegui/blob/1911455622d537a22b16778f2b7d244d526f125c/nicegui/elements/interactive_image.js#L14-L17

**Suggestion:**

Add a way to change the stroke color of the cross hair.

**Comment by rodja** [comment]
Yes, that would be awesome.
